reflex/constants.py

```diff
@@ -65,10 +65,14 @@ def get_value(key: str, default: Any = None, type_: Type = str) -> Type:
 # Bun config.
 # The Bun version.
 BUN_VERSION = "0.7.0"
+# Min Bun Version
+MIN_BUN_VERSION = "0.7.0"
```

Comment:
We still need a floor bun version for custom bun path use cases. If we increase the current bun version, users won't necessarily need to update bun versions(which could be annoying)